financial-connections/src/main/java/com/stripe/android/financialconnections/ui/FinancialConnectionsSheetNativeActivity.kt

```diff
@@ -0,0 +1,58 @@
+package com.stripe.android.financialconnections.ui
+
+import android.os.Bundle
+import androidx.activity.compose.setContent
+import androidx.appcompat.app.AppCompatActivity
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.ui.Modifier
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.rememberNavController
+import com.airbnb.mvrx.viewModel
+import com.stripe.android.financialconnections.navigation.NavigationDirections
+import com.stripe.android.financialconnections.presentation.FinancialConnectionsSheetNativeViewModel
+
+internal class FinancialConnectionsSheetNativeActivity : AppCompatActivity() {
+
+    val viewModel: FinancialConnectionsSheetNativeViewModel by viewModel()
```

Comment:
IUUC at some point a web page is displayed through CustomTab, in which case a different Activity is opened on top of this one, if it's the case, it's possible that `FinancialConnectionsSheetNativeActivity` will be destroyed by OS and recreated(you can force this by choosing don't keep activities in developer options). 

If that happens this Activity and the dagger graph `FinancialConnectionsSheetNativeViewModel` in will get recreated, including the subcomponent provider saved within - just want to confirm would that impact the creation of `ConsentViewModel`  or the recreation of the `FinancialConnectionsSheetNativeActivity` ?

Reply:
Yeah, this is contemplating activity recreation and process kill scenario. 

- Configuration changes (activity dies, ViewModel survives) -> state lives entirely in the ViewModel, no impact on any layer.
-Process kills (ViewModel gets re-instanced) All the dependencies provided via Intent from the SDK consumer (such as `FinancialConnectionsSheet#Configuration`) are bound when building the corresponding Component / Subcomponent via @BindsInstance. We also persist fields we don't want to re-fetch after process kills using `SavedHandleState`. Thanks to Mavericks, that's done transparently using [@PersistState](https://airbnb.io/mavericks/#/persist-state).

Reply:
Also, `navigation-compose` is built with process kills in mind. Current screen and back stack are also persisted behind the scenes:

![image](https://user-images.githubusercontent.com/99293320/172730467-03bf3485-9dfb-4747-936a-26af56def4c7.png)

Reply:
sg, thanks for the exaplain!